baseMdl/mdl.py

```diff
@@ -85,76 +41,14 @@ def discretize(obs, lower_bounds, upper_bounds, buckets):
         return tuple(new_obs)
 
     def train_from_scratch(self):
-        """Train new model with brand new q_table"""
-        q_table = np.zeros(self.buckets + (len(self.env.action_space),))
-
-        episodes = 501
-
-        alpha = 0.1
-        gamma = 0.9
-
-        # exploration
-        epsilon = 1.0
-        """ min_epsilon = 0.1
-        max_epsilon = 1.0
-        decay = 0.01 """
-
-        print(f"Training model from scratch for {episodes - 1} episodes...")
-        for episode in range(episodes):
-            current_state = self.discretize(
-                self.environment.reset(pygame.HIDDEN),
-                self.lower_bounds,
-                self.upper_bounds,
-                self.buckets,
-            )
-            while self.environment.running:
-                total_reward = 0
-
-            epsilon = self.min_epsilon + (self.max_epsilon - self.min_epsilon) * np.exp(
-                -self.decay * episode
-            )
-
-            while self.env.running:
-                exp_tradeoff = random.uniform(0, 1)
-
-                if exp_tradeoff > epsilon:
-                    action = np.argmax(q_table[current_state])
-                else:
-                    action = random.choice(self.env.action_space)
-
-                observation, reward, done = self.env.step(action)
-
-                new_state = self.discretize(
-                    observation, self.lower_bounds, self.upper_bounds, self.buckets
-                )
-
-                total_reward += reward
-
-                self.q_table[current_state][action] += alpha * (
-                    reward
-                    + gamma * np.max(self.q_table[new_state])
-                    - self.q_table[current_state][action]
-                )
+        pass
 
-                current_state = new_state
-
-                # env.render()
-
-                if done:
-                    self.env.running = False
-
-            if not episode % 100:
-                print(
-                    f"Episode: {episode} | Reward: {total_reward} | Epsilon: {epsilon}"
-                )
-
-        return q_table
-
-    def import_model():
+    def import_model(self):
         while True:
             mdl_file = input("Enter model file name (.npy): ")
```

Comment:
Aquí algo similar a lo de ask_to_save, donde se busca en el directorio de savedModels para leer el modelo que se quiere observar:
```suggestion
            mdl_file = input("Enter model file name (.npy): ")
             # Gets the path to the savedModels directory
            models_dir = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), self.models_dir_name)
            # Gets the path to the saved model file
            mdl_file = os.path.join(models_dir, mdl_file)
```